src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx

```diff
@@ -44,9 +46,10 @@ const LiveRadioContainer = ({ idAttr, externalId, id }) => {
   );
 
   const embedSource = embedUrl({
-    requestUrl: `${serviceId}/${id}/${lang}`,
+    mediaId: `${serviceId}/${id}/${lang}`,
     type: 'media',
     isAmp,
+    pageUrl: `${location.pathname}${location.search}`,
```

Comment:
I think I remember you mentioning this isn't actually url or something. Any chance we could change the name to something more descriptive of what it is then?

Reply:
According to the internets, the different parts of a URL are:
protocol -> https
host -> www.bbc.co.uk
pathname -> pidgin
parameters -> renderer_env=live

If for example the full URL was https://www.bbc.com/pidgin?renderer_env=live, then `${location.pathname}${location.search}` -> `pidgin?renderer_env=live`

Would `urlPath` work? Or simply `url`?

Reply:
We have reworked this completely, and are only passing queryString instead of the entire path